src/okapi_em/_widget.py

```diff
@@ -230,15 +231,15 @@ def ui_do_tab_ResMeas(self):
             vBox4.addLayout(qgrid3_layout)
             qgrid3_layout.addWidget(QLabel("tile size (px):"),0,0)
             self.spbQuolTileSize_px = QSpinBox()
-            self.spbQuolTileSize_px.setMinimum(8)
+            self.spbQuolTileSize_px.setMinimum(128)
             self.spbQuolTileSize_px.setMaximum(10000)
             self.spbQuolTileSize_px.setValue(256)
             qgrid3_layout.addWidget(self.spbQuolTileSize_px, 0,1)
             
             #add pixel size box
             qgrid3_layout.addWidget(QLabel("pixel size (nm):"),1,0)
             self.spbQuolPixelSize_nm = QDoubleSpinBox()
-            self.spbQuolPixelSize_nm.setMinimum(0.001)
+            self.spbQuolPixelSize_nm.setMinimum(1e-6)
```

Comment:
Was a just in case, so if we decide to make the pixel size field more general and allow users to specify units then this would at least make sure it's not zero